packages/webpack-cli/lib/utils/unknown-args.js

```diff
@@ -1,5 +1,25 @@
-const commandNames = require('./commands').names;
-const flagNames = require('./core-flags').names;
+const { commands, core } = require('./cli-flags');
+
+// Contains an array of strings with commands and their aliases that the cli supports
+const commandNames = commands
+    .map(({ alias, name }) => {
+        if (alias) {
+            return [name, alias];
+        }
+        return [name];
+    })
+    .reduce((arr, val) => arr.concat(val), []);
+
+// Contains an array of strings with core cli flags and their aliases
+const flagNames = core
+    .map(({ alias, name }) => {
+        if (name === 'help') return [];
+        if (alias) {
+            return [`--${name}`, `-${alias}`];
+        }
+        return [`--${name}`];
+    })
+    .reduce((arr, val) => arr.concat(val), []);
```

Comment:
Ideally we should avoid it, because we do it on `require` and reduce performance in initial start, because we have very many flags, but I will fix it in future too

Reply:
yep will improve it next